scripts/create_makefile.rb

```diff
@@ -1,6 +1,6 @@
 require 'fileutils'
 ABS_ROOT = FileUtils.pwd
-CMOCK_DIR = File.expand_path(ENV.fetch('CMOCK_DIR', File.join(ABS_ROOT, '..', '..')))
+CMOCK_DIR = File.expand_path(ENV.fetch('CMOCK_DIR', File.join(ABS_ROOT, '..')))
```

Comment:
This seems a bit unrelated - or am I wrong?

Reply:
true, that's probably what broke CI as well. For local testing i did `cd scripts ; ruby create_makefile.rb ...` Apparently usage within framework is different...

 Reverted this change now